### PR TITLE
Update GSoC org admin guide with the new announcements

### PR DESCRIPTION
**Pull request by aliok**
Update GSoC org admin guide with the new announcements

### CODE REVIEW

File: programs/program-statistics.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
 # Program Statistics
 
+These are the number of successful internships per year for each program.
```
Comment (aliok) [inline programs/program-statistics.md:3]: Are these actually mentees that passed? Just assumed they are.

Would we want to also list number of passed mentees / failed mentees broken down?
We need to check if Google / LFX publishes break down of passes/failures. If they don't, maybe we shouldn't do the break down and simply have the numbers we have right now.

File: programs/program-statistics.md [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
+
 | Year | Program                          | Internships | Total (per year) |
-| ---- | -------------------------------- | ----------- | ---------------- |
-| 2022 | LFX Mentorship (CommunityBridge) | TBD         | TBD              |
+|------|----------------------------------|-------------|------------------|
+| 2023 | LFX Mentorship                   | TBD         | TBD              |
+|      | GSoC                             | TBD         |                  |
+| 2022 | LFX Mentorship                   | TBD         | TBD              |
 |      | GSoC                             | TBD         |                  |
```
Comment (aliok) [inline programs/program-statistics.md:10]: Do you know the numbers for 2022 @nate-double-u ?

File: programs/program-statistics.md
Comment (aliok) [inline programs/program-statistics.md:1]: What's the purpose of this page?

Based on the purpose, I say we either improve it, or kill it.

File: programs/program-statistics.md (hunk 2 above)
Comment (nate-double-u) [reply programs/program-statistics.md:10]: We can find out 🙂. Related to #1000

File: programs/program-statistics.md
Comment (nate-double-u) [reply programs/program-statistics.md:1]: I think need to improve it -- I've had requests to both improve it and highlight it. (see #1000)

**Review by nate-double-u — APPROVED** [review]
Thanks for this update @aliok

**Pull request merged**